src/Controller/Admin/DataObject/ClassController.php

```diff
@@ -1256,7 +1256,13 @@ public function objectbrickTreeAction(Request $request, EventDispatcherInterface
             }
         }
 
+        usort($groups, function ($a, $b) {
+            return strcmp($a['text'], $b['text']);
+        });
```

Comment:
Shouldn't the spaceship be enough here?

```suggestion
        usort($groups, static fn ($a, $b) => $a['text'] <=> $b['text']);
```

Reply:
Yes, absolutely, thanks!